README.md

```diff
@@ -1,30 +1,16 @@
-# wise-agents
+# wise-agents a distributed multi AI agent framework
```

Comment:
For readability, maybe "Wise Agents: A Distributed AI Multi-Agent Framework" would be better.